homeassistant/components/recorder/models.py

```diff
@@ -28,6 +28,8 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+DB_TIMEZONE = "+00:00"
```

Comment:
Should this be `Z` to indicate it's UTC?

```suggestion
DB_TIMEZONE = "Z"
```

Reply:
Z is cleaner if it works.  Will test

Reply:
Works